src/test/java/gov/nj/innovation/customAwsIdp/lambda/GetSamlResponseHandlerTest.java

```diff
@@ -39,12 +43,12 @@ public class GetSamlResponseHandlerTest {
      */
     @SystemStub
     private final EnvironmentVariables envVariables = new EnvironmentVariables(
-            "KEY_PRIVATE_EXPONENT", "123",
-            "KEY_PRIME_P", "123",
-            "KEY_PRIME_Q", "123",
-            "KEY_PRIME_EXPONENT_P", "123",
-            "KEY_PRIME_EXPONENT_Q", "123",
-            "KEY_CRT_COEFFICIENT", "123",
+            "KEY_PRIVATE_EXPONENT_NAME", "custom-aws-idp-private-key-private-exponent",
+            "KEY_PRIME_P_NAME", "custom-aws-idp-private-key-prime-p",
+            "KEY_PRIME_Q_NAME", "custom-aws-idp-private-key-prime-q",
+            "KEY_PRIME_EXPONENT_P_NAME", "custom-aws-idp-private-key-prime-exponent-p",
+            "KEY_PRIME_EXPONENT_Q_NAME", "custom-aws-idp-private-key-prime-exponent-q",
+            "KEY_CRT_COEFFICIENT_NAME", "custom-aws-idp-private-key-crt-coefficient",
```

Comment:
This block sets the environment variables which are seen during the run of the test. Since the environment variables are just hardcoded strings without security concerns, I just copy-pasted them here from the serverless.yml file. The code used to expect the env vars to be our secret integers, but now it expects the parameter names which are defined in SSM as the keys for those secret integer values. The stubbing at the bottom of this file will return "123" when SSM is queried for these keys.

All good? Unclear?